lib/utils/positionFns.js

```diff
@@ -103,12 +103,13 @@ export function createCoreData(draggable: DraggableCore, x: number, y: number):
 // Create an data exposed by <Draggable>'s events
 export function createDraggableData(draggable: Draggable, coreData: DraggableData): DraggableData {
   const scale = draggable.props.scale;
+  const angleRadian = Math.PI * draggable.props.angle / 180.0;
   return {
     node: coreData.node,
-    x: draggable.state.x + (coreData.deltaX / scale),
```

Comment:
This is nice; however the vast majority of usage of this library will have `angle: 0`, and I don't want to add overhead to every user just for this edge case. Can you clean this up so the code only runs if `draggable.props.angle` is 0?